### ISSUE TITLE
Incorrect Js being generated

### ISSUE THREAD

**Issue by nxproject** [trullock/NUglify#124]
If you look at Xhr.js line 860 and compare that to cXhr.js column 1157, you will see the generated code, except the code at cXhr.js that precedes it comes from Xhr.js line 223.

I may have other issues, but the cascade from this one makes things unclear.

Thank you for your time.

[Issues.zip](https://github.com/trullock/NUglify/files/5091628/Issues.zip)

**Comment by trullock** [comment]
I'm gonna need a much more succinct example, with input, expected output, actual output and NUglify parameters.

What are you saying is wrong?

**Comment by nxproject** [comment]
I do not believe that I can come up with a succinct example, as I believe that the complexity of the JavaScript code is what is causing it.   I have tested other minifiers and they all seem to work fine, so I am closing this.  Thank you for your time.

**Comment by trullock** [comment]
If theres an issue I'm keen to understand it.

Can you at least tell me what you think is wrong?

**Comment by nxproject** [comment]
The logic mapped calls to __checkDisposed into calls to __initNativeXhr.

That is the first error, as there seem to be more as the mnfied code that NUglify is producing is totally different from other minifiers.

FYI:  This may be happening in a handful of the 600+ scripts that I have minfied.  As I could not trust the NUglify output and due to the dynamic nature of the project that I am working on, I had to switch to another minifier so it would be difficult to reproduce all instances.

Again, thank you for your time.
